composer.json

```diff
@@ -11,14 +11,8 @@
             "homepage": "http://www.akeneo.com"
         }
     ],
-    "repositories": [
-        {
-            "type": "vcs",
-            "url": "https://github.com/akeneo/pim-community-dev.git"
-        }
-    ],
     "require": {
-        "akeneo/pim-community-dev": "^2.2"
+        "akeneo/pim-community-dev": "dev-master"
```

Comment:
Shouldn't we wait for the 3.0 branch? It should be done this afternoon.

Reply:
I'll make the change when tag is ready, but with this the community can already use it with master